__tests__/app.test.js

```diff
@@ -85,3 +85,26 @@ describe('GET /api/articles/:article_id', () => {
   //     });
   // });
 });
+describe('GET /api/articles', () => {
+  test.only('200: Responds with all articles in database', () => {
```

Comment:
remove .onlys before pushing otherwise we can't be sure all your tests pass - future prs will be rejected if they include any .onlys